scss/components/_case-info.scss

```diff
@@ -0,0 +1,10 @@
+.___outside-namespace {
+  .prospect__case-info {
+    background-color: $crm-white;
+    border-radius: 4px;
+    padding: 5px;
+    position: absolute;
+    right: 24px;
+    top: 53px;
```

Comment:
can't these values be represented using variables? Do we have access to the Shoreditch variables here?

Reply:
Used variables for `border-radius` and `padding`. For rest of the values we dont have existing variables. and they wont be reused, so I dont think we should create new variables.